apollo-compiler/src/main/kotlin/com/apollographql/apollo/compiler/SchemaTypeSpecBuilder.kt

```diff
@@ -433,18 +449,18 @@ class SchemaTypeSpecBuilder(
   }
 
   private fun inlineFragmentsResponseMapperSpec(nameOverrideMap: Map<String, String>,
```

Comment:
`codestyle`: Looks like here, it would be better to put the first param in a new line